app/views/devise/sessions/new.html.erb

```diff
@@ -13,7 +13,7 @@
   </div>
 
   <div class="form-actions">
-    <%= f.button :submit, "Log in" %>
+    <%= f.button :submit, "Log in" %>s
```

Comment:
@nfarinello why is there an s at the end of this line?

Reply:
done